scripts/benchmark.cr

```diff
@@ -105,5 +105,6 @@ results << benchmark { image_rgba.crop!(200, 200, 100, 100) }
 results << benchmark { image_rgba.gaussian_blur!(10) }
 results << benchmark { image_rgba.horizontal_blur!(10) }
 results << benchmark { image_rgba.vertical_blur!(10) }
+results << benchmark { image_rgba.pad!(100) }
```

Comment:
I didn't want to update the `BENCHMARKS.md` with my potato of a machine's specs :P ~2 mb memory and ~250 microseconds for it to run for me.

Reply:
No worries, I'll update them when releasing the next version.